python/pip_install/pip_repository.bzl

```diff
@@ -542,6 +544,22 @@ def _whl_library_impl(rctx):
     if not rctx.delete("whl_file.json"):
         fail("failed to delete the whl_file.json file")
 
+    if rctx.attr.whl_patches:
+        patches = {}
+        for patch_file, json_args in patches.items():
```

Comment:
This loop never does anything, given the previous line, no?

Reply:
Yes, see #1637